task_storing_groceries/input_values/pose.txt

```diff
@@ -0,0 +1 @@
+{'header': {'frame_id': 'base_link'},'pose': {'position': {'x': 0.5,'y': 0,'z': 0.4},'orientation': {'w': 1}}}
```

Comment:
How are these used? These files look like the inputs to the task state machine, so a usage note in the readme seems appropriate.